parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoRecordConverter.java

```diff
@@ -45,14 +48,13 @@ public void add(Object a) {
     }
   }
 
-
-  public ProtoRecordConverter(Class<? extends Message> protoclass, MessageType parquetSchema) {
-    super(new SkipParentValueContainer(), protoclass, parquetSchema);
+  public ProtoRecordConverter(Configuration conf, Class<? extends Message> protoclass, MessageType parquetSchema, Map<String, String> extraMetadata) {
```

Comment:
These two constructors are public in a public class. Can we mark the old constructors deprecated and keep them? Despite (AFAIK) this class is not supposed to be part of the public API, unfortunately it isn't package private, and clients could use it: a change like this could break their code. Though this scenario is unlikely, unfortunately we faced similar issues before.

Reply:
It sounds fair. I'll keep deprecated constructors for now.